### ISSUE TITLE
Ticks of contour plots with PyPlot

### ISSUE THREAD

**Issue by y-nkt** [JuliaPlots/Plots.jl#1881]
I found a strange behavior on ticks of contour plot with PyPlots.

Default setting does not make ticks:
```
using Plots; pyplot()
contour(-2:0.1:2,-2:0.1:2,(x,y)->x^2+y^2, fill=true, levels=50)
```
![contour1](https://user-images.githubusercontent.com/6076710/50761556-080a1a00-12ae-11e9-915c-7e21d4d3ef41.png)

If I put a grid option, I get ticks:
```
`using Plots; pyplot()
contour(-2:0.1:2,-2:0.1:2,(x,y)->x^2+y^2, fill=true, levels=50, grid=:none)
```
![contour2](https://user-images.githubusercontent.com/6076710/50761560-0ccece00-12ae-11e9-8604-8217768e2e6e.png)

It seems better to use the second case as default.
Thank you for your development.

**Comment by isentropic** [comment]
Ticks are there if you zoom out. Somehow the image gets drawn on top of ticks
![image](https://user-images.githubusercontent.com/24591123/80998635-3dad1c00-8e7e-11ea-8359-13225f997686.png)

**Comment by ashwani-rathee** [comment]
Hey @BeastyBlacksmith ,is this issue still open and something to work upon??

**Comment by LakshyaKhatri** [comment]
Hello @isentropic, I looked into the codebase and understood that I might have to edit files for each backend separately. Please let me know if I am doing the right thing or not. Any pointers might help in the latter case.

**Comment by isentropic** [comment]
Yes I believe, this should be handled for all backends, perhaps some backends handle this already.

In pyplot, this issue would be solved by looking into `zorder` of axis [ticks](https://matplotlib.org/3.2.1/api/_as_gen/matplotlib.axes.Axes.tick_params.html) or [this](https://matplotlib.org/3.1.1/gallery/images_contours_and_fields/contourf_demo.html#sphx-glr-gallery-images-contours-and-fields-contourf-demo-py)

**Comment by isentropic** [comment]
The issue with pyplot is that, layer order of axis ticks is below the layer of contour. This way contour ends up above the ticks.
Perhaps this line is what yuo are looking for https://github.com/JuliaPlots/Plots.jl/blob/47590b2538b9d0387b14d46eb49e7257f904dde6/src/backends/pyplot.jl#L850
